05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,43 +1,36 @@
-.lawn {
-    width: 100%;
-    height: 300px;
+.big {
+  width:500px;
+  float:left;
 }
 
-.lawn h1 {
-    text-align: center;
-    font-family: monospace;
-    color: darkgray;
+img {
+  height:100%;
+  width:100%;
 }
 
-.lawn {
-    background-image: url(../img/banner-garden-stmichaelsmount.jpg);
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: cover;
+#ipsum {
+  width:500px;
+  float:right;
```

Comment:
remember, if you float things the same direction, they'll stack next to each other!